client/a8c-for-agencies/sections/sites/features/jetpack/site-details-pane/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import AgencySiteTags from 'calypso/a8c-for-agencies/components/agency-site-tags';
+import SiteTagType from 'calypso/a8c-for-agencies/types/site-tag';
+import './style.scss';
+
+interface Props {
+	site: any;
+}
+
+export function SiteDetailsPane( { site }: Props ) {
```

Comment:
I think we can remove this component in favor of SiteDetails in the a4a features folder.